### PR TITLE
Chapter 4

### PR DESCRIPTION
**Pull request by duongtruong9294**
chapter_4

### CODE REVIEW

**Comment by duongtruong9294** [comment]
ready

File: app/assets/javascripts/static_page.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (hhhhuy112) [inline app/assets/javascripts/static_page.coffee]: remove this file if you don't use it

File: app/assets/stylesheets/static_page.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPage controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (hhhhuy112) [inline app/assets/stylesheets/static_page.scss]: remove this file if you don't use it

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -1,4 +1,12 @@
 # frozen_string_literal: true
 
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = "Ruby on Rails Tutorial Sample App"
```
Comment (hhhhuy112) [inline app/helpers/application_helper.rb]: I18n for "Ruby on Rails Tutorial Sample App"

**Comment by hhhhuy112** [comment]
commented

**Comment by duongtruong9294** [comment]
ready

File: app/helpers/static_page_helper.rb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+module StaticPageHelper
+end
```
Comment (chien-pv) [inline app/helpers/static_page_helper.rb]: Xóa đi nếu không dùng

**Comment by chien-pv** [comment]
commented

**Comment by duongtruong9294** [comment]
ready

**Comment by hhhhuy112** [comment]
merged
You can merge it

**Pull request merged**